roles/routeros_common/molecule/default/Vagrantfile

```diff
@@ -40,11 +61,13 @@ Vagrant.configure("2") do |config|
       # Disable DHCP client and force address to known address (inventories/test/routeros_common.py)
       node.vm.provision "routeros_command", name: "Disable DHCP client on host_only", command: "/ip dhcp-client disable [find interface=host_only]"
       node.vm.provision "routeros_command", name: "Add IP address", command: "/ip address add interface=host_only address=#{vm['ip']}/24"
-      # Upload and execute a setup script file
-      node.vm.provision "routeros_file", name: "Upload initial common configuration", source: "provision/common_setup.rsc", destination: "common_setup.rsc"
-      node.vm.provision "routeros_command", name: "Exec custom script", command: "/import common_setup.rsc", check_script_error: true
-      node.vm.provision "routeros_file", name: "Upload initial host configuration", source: "provision/#{vm['name']}.rsc", destination: "host_setup.rsc"
-      node.vm.provision "routeros_command", name: "Set host IP", command: "/import host_setup.rsc", check_script_error: true
+
+      # Upload and execute a common script file
+      node.vm.provision "routeros_file", name: "Upload initial common configuration", source: "provision/common.rsc", destination: "common.rsc"
+      node.vm.provision "routeros_command", name: "Exec custom script", command: "/import common.rsc", check_script_error: true
+      # Upload and execute a test specific script file
+      node.vm.provision "routeros_file", name: "Upload initial host configuration", source: "provision/#{vm["name"].sub(/ros.\./, '')}.rsc", destination: "host.rsc"
```

Comment:
matter of opinion, so feel free to disregard: the regex sub here is a little subtle. an explicit mapping from vm to provisioning file would be more explicit and easier to read. I probably spent several minutes confused about how ros6.incorrect gets set up differently than ros6.correct, etc.

as written you're kind of packing a lot of implicit logic into this magical vm name instead of treating that as an opaque string. that always strikes me as a little suspicious.

Reply:
I think I have a better solution.  Have a look.